feature/platform/tests/optics_power_and_bias_current_test/optics_power_and_bias_current_test.go

```diff
@@ -30,29 +30,16 @@ import (
 )
 
 const (
-	ethernetCsmacd         = oc.IETFInterfaces_InterfaceType_ethernetCsmacd
-	transceiverType        = oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_TRANSCEIVER
-	sleepDuration          = time.Minute
-	minOpticsPower         = -40.0
-	maxOpticsPower         = 10.0
-	minOpticsHighThreshold = 1.0
-	maxOpticsLowThreshold  = -1.0
+	ethernetCsmacd  = oc.IETFInterfaces_InterfaceType_ethernetCsmacd
+	transceiverType = oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_TRANSCEIVER
+	minOpticsPower  = -40.0
+	maxOpticsPower  = 10.0
 )
 
 func TestMain(m *testing.M) {
 	fptest.RunTests(m)
 }
 
-// Topology:
-//   ate:port1 <--> port1:dut:port2 <--> ate:port2
-//
-//  Sample CLI command to get telemetry using gmic:
```

Comment:
Any reason removing this comment? Seems helpful. 

nit: yes you can update the link to: https://github.com/openconfig/gnmic

Reply:
I think it may be more relevant for README? I.e. that's just a description on how to do gnmi call manually, and there's nothing specific about it?
And topology part above is just incorrect i think, we don't use port2 for example.